ghmirror/data_structures/monostate.py

```diff
@@ -60,6 +63,9 @@ def __setitem__(self, key, value):
     def __iter__(self):
         return iter(self._data)
 
+    def __sizeof__(self):
+        return sys.getsizeof(self._data)
```

Comment:
I don't know if this is working. When I run the server locally and check the endpoint, I am getting the same size constantly. "github_mirror_cache_size 264.0"

If I do something like below, I get a better estimate. Obviously this is bad because its O(n) so I am not suggesting we do it, just illustrating. 

`    def __sizeof__(self):
        total_size = 0
        for item in self._data:
            total_size += sys.getsizeof(item)
        return total_size`

I think the issue is that sys.getsizeof is not capturing all the objects contents, see [https://docs.python.org/3/library/sys.html#sys.getsizeof](here).

Reply:
There are 2 aspects here:
- the dictionary is resized only when needed. Python uses the 70% occupacy threshold to resize the dictionary, so it's normal that it takes multiple insertions before you see it growing.
- you're right, even though we are corretly returning the dictionary size, we are not compupting the size of its items. Great catch! On your suggestion, you're computing the keys size. The missing part is to compute also the size of the values. Sending an update soon :)

Reply:
sounds good!